content/Watsonx/Assistant/103.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: '103: Out-the-box AI'
 timeToComplete: 30
-updated: 2023-03-31
+updated: 2024-02-06
 ---
 
 # 103: Out-the-box Artificial Intelligence
```

Comment:
Use the following text for quiz alerts. 

Heads Up! Quiz material will be flagged like this!